### ISSUE TITLE
Move Cloud "Login entry point" to the sidebar of IDE2

### ISSUE THREAD

**Issue by davegarthsimpson** [arduino/arduino-ide#1877]
### Describe the request

We hope to introduce and promote Arduino Cloud to as many IDE2 users as possible. We would therefore like to make the integrated cloud login in IDE2 more visible.

#### Outcome

A user avatar together with a popover menu should be added to the "first level of the interface", in the sidebar. The popover menu should contain Arduino cloud related actions, namely Sign in/out and links to the cloud platform.

The existing avatar should be removed, and the "Cloud account state element" (at the bottom of the cloud sketchbook list) should be augmented as per design spec. 

### Additional context
Figma links here

### Issue checklist

- [X] I searched for previous requests in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the feature was still missing when using the latest [nightly build](https://www.arduino.cc/en/software#nightly-builds)
- [X] My request contains all necessary details

**Comment by kittaakos** [comment]
The new requested _login entry point_  occupies the space from the default [_Account management_](https://code.visualstudio.com/updates/v1_44#_account-management) in Theia/VS Code.

From VS Code:

<img width="104" alt="Screen Shot 2023-02-10 at 14 57 24" src="https://user-images.githubusercontent.com/1405703/218109675-6c42ac78-2125-4e9c-93d5-1dfa7e5c7d77.png">

It will likely break the IDE2 UI if any future VS Code extension contributing to [`authentication`](https://code.visualstudio.com/api/references/vscode-api#authentication) API is added to IDE2. See the Theia logic [here](https://github.com/eclipse-theia/theia/blob/371ab02dda5c6df2c1381ab68b2f5ae341f7d258/packages/core/src/browser/common-frontend-contribution.ts#L456-L458). Currently, no extensions contribute to this API, so there is no UI issue now.

**Comment by kittaakos** [comment]
When `Advanced` > `Show/Hide Remote Sketchbook` is off (`"arduino.cloud.enabled": false`), then IDE2 does not show `File` > `New Remote Sketch`, and the _Remote Sketchbook_ is also hidden from the _Sketchbook_ view.

<img width="171" alt="Screen Shot 2023-02-10 at 15 07 17" src="https://user-images.githubusercontent.com/1405703/218111863-6bb40b20-e16d-40c1-84cb-3f80696cadaf.png">
<img width="255" alt="Screen Shot 2023-02-10 at 15 07 23" src="https://user-images.githubusercontent.com/1405703/218111873-9e22554d-0f99-413f-8e57-e00670d10c01.png">


@91volt, does IDE2 want to hide the _login entry point_ from the sidebar? Thank you!

**Comment by davegarthsimpson** [comment]
> The new requested _login entry point_ occupies the space from the default [_Account management_](https://code.visualstudio.com/updates/v1_44#_account-management) in Theia/VS Code.
> 
> From VS Code:
> 
> <img alt="Screen Shot 2023-02-10 at 14 57 24" width="104" src="https://user-images.githubusercontent.com/1405703/218109675-6c42ac78-2125-4e9c-93d5-1dfa7e5c7d77.png">
> 
> It will likely break the IDE2 UI if any future VS Code extension contributing to [`authentication`](https://code.visualstudio.com/api/references/vscode-api#authentication) API is added to IDE2. See the Theia logic [here](https://github.com/eclipse-theia/theia/blob/371ab02dda5c6df2c1381ab68b2f5ae341f7d258/packages/core/src/browser/common-frontend-contribution.ts#L456-L458). Currently, no extensions contribute to this API, so there is no UI issue now.

good point, @91volt if in the future we add something that contributes to the `authentication` api we may need to consider moving the cloud avatar to avoid obscuring the default Theia/VS Code element.
